features.org

```diff
@@ -89,113 +57,134 @@ List of [[https://orgmode.org/worg/org-contrib/babel/languages.html][supported l
 
 ** Description
 
-Org allows you to fold sections into an outline for graceful navigation, then
-seamlessly expand individual sections to read and edit using the =tab= key.
+The fundamental structure of an Org document is a tree, with nested,
+collapsible sections forming its backbone.  You can fold and unfold
+whole trees with single keystroke -- =tab=.
 
-You can easily navigate through headlines and (un)fold (sub)sections of
-your Org documents.
+Org's trees make for quick, graceful navigation and powerful
+information hiding that lets you focus on the task at hand without
+compromising the ability to get the information you need complete it.
 
-*** More
+Controlling section visibility is only a tiny part of Org's set of
+tools and commands for structured editing of plain text.  Org mode
+leverages Org's regular syntax to transform otherwise tedious tasks
+into single key commands, freeing you to focus on content rather than
+form.
 
-More on [[file:manual/Document-Structure.html][editing and document structure]] (manual)
+*** More
 
-* Task Management
+More on [[file:manual/Document-Structure.html][document structure]] (manual)
+* Transparent Tables
   :PROPERTIES:
-  :CUSTOM_ID: planning
+  :CUSTOM_ID: tables
   :HTML_CONTAINER_CLASS: side-fig
   :END:
 
-#+attr_html: :class main-image :alt Demonstration of planning with org, managing a scheduled todo list
-[[file:resources/img/features/planning.gif]]
+#+attr_html: :class main-image :alt Manipulating tables with Org, moving cells/rows and executing formulas
+[[file:resources/img/features/tables.gif]]
 
 ** Description
 
-Org is an extremely capable task manager. "Org" is short for "organisation"
-after all.
-
-Turn sections into /tasks/ by adding keywords such as =TODO= representing the state of that task.
-Switch states easily with the keyboard shortcut =S-<left/right>=.
-
-See your tasks in a planner (called an /Agenda/ in Org) by adding timestamps to them.
+Org includes a powerful table editor.
 
-All done with just plain text markup.
+=tab= moves between columns. =return= moves between rows.
```

Comment:
Maybe turn this into one sentence? "=tab= moves between columns, and =return= moves between rows."
Just a thought.

Oh, and should we maybe use =TAB= and =RET= ?

Reply:
I think that including `, and` in this context makes it harder to understand what we are trying to convey, because it forces the sentence to have two subjects, rows and columns, rather than one subject per sentence which is easier to parse.

The Emacs manual uses lowercase https://www.gnu.org/software/emacs/manual/html_node/elisp/Function-Keys.html.

Reply:
Seems reasonable